v2g-liberty-cards/src/charger-settings-card.ts

```diff
@@ -93,6 +94,35 @@ export class ChargerSettingsCard extends LitElement {
     `;
   }
 
+  private _isV2GlibertyLoadbalancerEnabled(): boolean {
+    const ql = this._quasarLoadbalancerLimit?.state.toLowerCase();
+    return !(
```

Comment:
Since this is repeated in the dialog, consider moving this to a utility function.

Reply:
Indeed, was my original plan but went for a "solution known to me already" ;-)